src/components/annotations/annotation_defaults.js

```diff
@@ -35,69 +35,56 @@ module.exports = function handleAnnotationDefaults(annIn, fullLayout) {
     var borderWidth = coerce('borderwidth');
     var showArrow = coerce('showarrow');
 
-    if(showArrow) {
-        coerce('arrowcolor', borderOpacity ? annOut.bordercolor : Color.defaultLine);
-        coerce('arrowhead');
-        coerce('arrowsize');
-        coerce('arrowwidth', ((borderOpacity && borderWidth) || 1) * 2);
-        coerce('ax');
-        coerce('ay');
-        coerce('axref');
-        coerce('ayref');
-
-        // if you have one part of arrow length you should have both
-        Lib.noneOrAll(annIn, annOut, ['ax', 'ay']);
-    }
-
     coerce('text', showArrow ? ' ' : 'new text');
     coerce('textangle');
     Lib.coerceFont(coerce, 'font', fullLayout.font);
 
     // positioning
-    var axLetters = ['x', 'y'];
+    var axLetters = ['x', 'y'],
+        arrowPosDflt = [-10, -30],
+        tdMock = {_fullLayout: fullLayout};
```

Comment:
Halloween is almost upon us, so maybe I'll scare you even *more* by swapping `td` for `gd` EVERYWHERE that remnant of streambed was lingering. EXORCISE THE DEMON! https://github.com/plotly/plotly.js/pull/1078/commits/5ccd0832beae3eda13d6968198a89a6c27d7f540